src/Widgets/AnimatedLogo/AnimatedLogo.tsx

```diff
@@ -0,0 +1,24 @@
+import { TouchableOpacity, View, Image } from "react-native"
+import React, { useEffect, useRef, useState } from 'react';
+import { BottomTabBarProps } from "@react-navigation/bottom-tabs";
+//@ts-ignore - the file exists
+import PEW_35_ANIMATED_LOGO from '../../../assets/Mobile-Persist-600px-GIF.gif';
+import { ROUTE_NAMES } from "../../TypesAndInterfaces/routes";
+import { StackNavigationProps } from "../../TypesAndInterfaces/custom-types";
+import { StackActions } from '@react-navigation/native';
+
+export const AnimatedLogo = (props:StackNavigationProps):JSX.Element => {
+
+    const [timeoutID, setTimeoutID] = useState<NodeJS.Timeout>(setTimeout(() => props.navigation.dispatch(StackActions.replace(ROUTE_NAMES.BOTTOM_TAB_NAVIGATOR_ROUTE_NAME)), 9000));
+
```

Comment:
Do we want to be a little tighter on the timing?
I timed the animation to be 5.6 seconds, so maybe a little past that.

Also is it possible to do a fade transition?  In the web I reduced the page opacity to zero, then swapped routes.

I like the tap through, that'll be helpful in by debugging.

Reply:
Changed timeout length to 7 seconds, which is the same as the portal. Also added a smooth fade transition.

Reply:
Perfect, thanks Man